.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -72,7 +72,7 @@ body:
     id: source
     attributes:
       label: >
-        What's the output of `git remote get-url origin; git rev-parse master; git rev-parse HEAD` ?
+        What's the output of `git remote get-url origin; git rev-parse main; git rev-parse HEAD` ?
```

Comment:
Since this part could differ based on the project (for example, Bazel itself still uses `master`), and `git rev-parse HEAD` is the more interesting question anyways -- could we just remove the middle part?

Reply:
Done.